src/messenger-shortcuts.js

```diff
@@ -171,7 +171,7 @@ function sendLike() {
 }
 
 function searchInConversation() {
-  var targetNode = getByAttr('em', 'data-intl-translation', 'Search in Conversation');
+  var targetNode = document.querySelector('._3szn._3szo ._5odt') // return the first one that matches the selector
   click(targetNode);
```

Comment:
I would recommend surrounding this call to `click()` with a null-check, since the info pane might not always be open!